migration/i9/oracle/202209011450_IDEMPIERE-5396.sql

```diff
@@ -0,0 +1,894 @@
+;
+
+-- Sep 1, 2022, 2:54:03 PM SGT
+INSERT INTO AD_ViewColumn (AD_Client_ID,AD_Org_ID,AD_ViewColumn_ID,AD_ViewColumn_UU,Created,CreatedBy,EntityType,IsActive,Updated,UpdatedBy,AD_ViewComponent_ID,ColumnName,ColumnSQL) VALUES (0,0,217518,'d557f631-462d-4589-8d49-b36752b7b7ff',TO_TIMESTAMP('2022-09-01 14:54:02','YYYY-MM-DD HH24:MI:SS'),100,'D','Y',TO_TIMESTAMP('2022-09-01 14:54:02','YYYY-MM-DD HH24:MI:SS'),100,200225,'M_Warehouse_ID','0')
+;
+
+-- Sep 1, 2022, 2:54:04 PM SGT
+INSERT INTO AD_ViewColumn (AD_Client_ID,AD_Org_ID,AD_ViewColumn_ID,AD_ViewColumn_UU,Created,CreatedBy,EntityType,IsActive,Updated,UpdatedBy,AD_ViewComponent_ID,ColumnName,ColumnSQL) VALUES (0,0,217519,'e2932715-861b-4bc8-a854-b2483f2efebb',TO_TIMESTAMP('2022-09-01 14:54:03','YYYY-MM-DD HH24:MI:SS'),100,'D','Y',TO_TIMESTAMP('2022-09-01 14:54:03','YYYY-MM-DD HH24:MI:SS'),100,200225,'IsSOTrx','r.issotrx')
+;
+
+-- Sep 1, 2022, 2:54:05 PM SGT
+INSERT INTO AD_ViewColumn (AD_Client_ID,AD_Org_ID,AD_ViewColumn_ID,AD_ViewColumn_UU,Created,CreatedBy,EntityType,IsActive,Updated,UpdatedBy,AD_ViewComponent_ID,ColumnName,ColumnSQL) VALUES (0,0,217520,'deb2deb9-0cde-478f-91bc-5744f0059d30',TO_TIMESTAMP('2022-09-01 14:54:04','YYYY-MM-DD HH24:MI:SS'),100,'D','Y',TO_TIMESTAMP('2022-09-01 14:54:04','YYYY-MM-DD HH24:MI:SS'),100,200225,'AD_Table_ID','660')
+;
+
+-- Sep 1, 2022, 2:55:42 PM SGT
+UPDATE AD_ViewColumn SET ColumnName='M_InOut_CreateFrom_v_ID',Updated=TO_TIMESTAMP('2022-09-01 14:55:42','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_ViewColumn_ID=217421
+;
+
+-- Sep 1, 2022, 2:56:07 PM SGT
+UPDATE AD_ViewColumn SET ColumnName='M_InOut_CreateFrom_v_ID',Updated=TO_TIMESTAMP('2022-09-01 14:56:07','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_ViewColumn_ID=217444
+;
+
+-- Sep 1, 2022, 2:56:23 PM SGT
+UPDATE AD_ViewColumn SET ColumnName='M_InOut_CreateFrom_v_ID',Updated=TO_TIMESTAMP('2022-09-01 14:56:23','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_ViewColumn_ID=217467
+;
+
+-- Sep 1, 2022, 2:56:45 PM SGT
+UPDATE AD_ViewColumn SET ColumnName='M_InOut_CreateFrom_v_ID',Updated=TO_TIMESTAMP('2022-09-01 14:56:45','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_ViewColumn_ID=217490
+;
+
+-- Sep 1, 2022, 2:57:08 PM SGT
+UPDATE AD_ViewColumn SET ColumnName='M_InOut_CreateFrom_v_ID',Updated=TO_TIMESTAMP('2022-09-01 14:57:08','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_ViewColumn_ID=217513
+;
+
+-- Sep 1, 2022, 2:58:28 PM SGT
+CREATE OR REPLACE VIEW M_InOut_CreateFrom_v(Q ty, Multiplier, C_UOM_ID, M_Locator_ID, M_Product_ID, C_Charge_ID, VendorProductNo, Line, C_OrderLine_ID, C_InvoiceLine_ID, M_RMALine_ID, C_BPartner_ID, C_Order_ID, C_Invoice_ID, M_RMA_ID, M_InOut_CreateFrom_v_ID, AD_Client_ID, AD_Org_ID, IsActive, UPC, M_Warehouse_ID, IsSOTrx, AD_Table_ID) 
```

Comment:
ERROR at line 1:
ORA-00907: missing right parenthesis

I think must be because there is a space after the Q